sql/schema/dsl.go

```diff
@@ -231,6 +231,12 @@ func NewView(name, def string) *View {
 	return &View{Name: name, Def: def}
 }
 
+// NewMaterialized creates a new materialized View.
+func NewMaterialized(name, def string) *View {
+	return NewView(name, def).
+		SetMaterialized(true)
```

Comment:
```suggestion
	return NewView(name, def).SetMaterialized(true)
```
Not really neede, just a suggestion.